apps/prairielearn/src/middlewares/redirectEffectiveAccessDenied.ts

```diff
@@ -55,4 +51,6 @@ export default function (err, req, res, next) {
 
   // give up, we couldn't figure out a useful redirect
   next(err);
-}
+};
+
+export default accessDeniedRedirect;
```

Comment:
Nit, conventionally we try to name these the same as the enclosing file, even if it's just default exported.